src/MenuGraphr.ts

```diff
@@ -721,6 +724,19 @@ export class MenuGraphr implements IMenuGraphr {
     }
 
     /**
+     * Clears saved indices that should be forgotten when the menu is deleted.
+     * 
+     * @param name   The name of the menu that is being deleted.
+     */
+    public clearMenuIndices(name: string): void {
+        const menu: IMenu = this.menus[name];
```

Comment:
Do you want this function to be public? As of now it is only referenced in MenuGraphr, I didn't think about making it private